swagger.yaml

```diff
@@ -2985,6 +2985,13 @@ paths:
         - userAuth: [admin, read-only-admin, user]
       description: Get minimum host requirements.
       operationId: GetHostRequirements
+      parameters:
```

Comment:
maybe instead of having another parameter you could add it `host-requirements`? you have master/worker/sno

Reply:
Why `in: header`?

Reply:
thought about it , but i was not sure what it break. So prefered smaller change but will open ticket for this one.

Reply:
You are adding parameters so it should not break anything

Reply:
I think that code will be simple as well.

Reply:
to say the truth i don't know who uses this api at all

Reply:
@avishayt what do you think?

Reply:
@tsorya @filanov I'm fairly sure this API is no longer used.  The new APIs are:
/clusters/{cluster_id}/host-requirements
/clusters/{cluster_id}/preflight-requirements

In both cases there should be no API change, just check if the cluster is SNO then return the appropriate requirements.